app/Observers/EntityObserver.php

```diff
@@ -197,6 +199,7 @@ public function updated(Entity $entity)
 
         // Queue job when an entity was updated
         EntityUpdatedJob::dispatch($entity);
+        EntityWebhookJob::dispatch($entity, $entity->campaign, auth()->user(), 2);
```

Comment:
Since campaign is taken from the entity, the whole param can be removed from the code and job